blog/2024-02-12-announcing-apache-pulsar-3-2.md

```diff
@@ -0,0 +1,80 @@
+
+### Null-key messages deletion during compaction
+
+Apache Pulsar supports [Topic Compaction](https://pulsar.apache.org/docs/3.2.x/concepts-topic-compaction/).
+
+In versions 3.1.0 and earlier of Pulsar, topic compaction retains messages that have null keys.
+
+Starting with Pulsar 3.2.0, the default behavior has been changed to delete messages with null keys during topic compaction, which helps in reducing storage costs. However, if necessary, you can revert to the previous behavior by setting `topicCompactionRetainNullKey=true` in `broker.conf`.
+
+For more details, refer to the description in [PIP-318](https://github.com/apache/pulsar/blob/master/pip/pip-318.md).
+
+### WebSockets new features
+
+The Pulsar WebSocket API now supports:
+- Consuming multiple topics as described in [PIP-307](https://github.com/apache/pulsar/blob/master/pip/pip_307.md).
+- End-to-end encryption, as described in [PIP-290](https://github.com/apache/pulsar/blob/master/pip/pip-290.md).
+
+### Enhanced security for Pulsar Functions & Pulsar IO
+
+Pulsar 3.2.0 provides increased security for Pulsar Functions & Pulsar IO by allowing users to configure Pulsar Connectors with non-plaintext secrets. More details in [PIP-289](https://github.com/apache/pulsar/blob/master/pip/pip-289.md) & the following pull requests:
+- [Make connectors load sensitive fields from secrets](https://github.com/apache/pulsar/pull/21675)
+- [Support configuration secret interpolation](https://github.com/apache/pulsar/pull/20901)
+
+### CLI user experience improvements
+
+This release introduces several handy features to the CLI, including:
+- [Add command line option for configuring the memory limit](https://github.com/apache/pulsar/pull/20663)
+- [Allow for deleting topics in bulk, using a regex or a file](https://github.com/apache/pulsar/pull/21664)
+- [Display the current cluster in `pulsar-admin clusters list`](https://github.com/apache/pulsar/pull/20614)
+
+### Build improvement
+
+Pulsar 3.2.0 also introduces enhancements to the build process and reliability, including a Bill of Materials (BOM) to simplify dependency management, as outlined in [PIP-326](https://github.com/apache/pulsar/blob/master/pip/pip-326.md). 
+
+### More
+
+Feel free to explore the [full release notes](https://github.com/apache/pulsar/releases/tag/v3.2.0) for a detailed list of all improvements and fixes.
+
+## Compatibility between releases
+
+When upgrading an existing Pulsar installation, it's crucial to perform component upgrades in a sequential manner.
+
+Starting from version 3.0, users have the option to perform live upgrades or downgrades between two consecutive LTS versions or two consecutive feature versions (which also include LTS versions).
+
+For the 3.2 series, you should be able to upgrade from version 3.1 or downgrade from the subsequently released version 3.3. If you are currently using an earlier version, please ensure that you upgrade to version 3.1 before proceeding further.
```

Comment:
> please ensure that you upgrade to version 3.1 before proceeding further.


I noticed in this [discussion](https://lists.apache.org/thread/objr5q14l4769pj6131fnf6y8vmnznbl) that it is possible to directly upgrade from 3.0 to 3.2, which seems to conflict with the information here that requires upgrading to 3.1 first.

Reply:
Thanks @crossoverJie you're right. I'll fix it